diff --git a/client/meson.build b/client/meson.build
--- a/client/meson.build
+++ b/client/meson.build
@@ -4,8 +4,8 @@ cc = meson.get_compiler('c')
 cppc = meson.get_compiler('cpp')

 use_fortran = false
-if get_option('with_fortran') or get_option('with_xtb') or get_option(
-    'with_cuh2',
+if get_option('with_fortran') or get_option(
+    'with_cuh2',
 )
     use_fortran = true
     add_languages('fortran', required: true)
@@ -326,10 +326,7 @@ endif
 if get_option('with_xtb')
     # Try finding system 'xtb'.
-    # If not found, look for 'xtb' in subprojects/ and use the 'xtb_dep_static' variable from it.
-    xtb_dep = dependency(
-        'xtb',
-        required: true,
-        fallback: ['xtb', 'xtb_dep_static'],
-    )
+    # Do not use fallback for conda-forge builds where xtb is a pre-built package.
+    xtb_dep = dependency('xtb', required: true)

     _deps += [xtb_dep]
@@ -433,4 +430,7 @@ if get_option('with_torch')
     lib_torch_list = ['c10', 'torch', 'torch_cpu', 'torch_global_deps']
     tdeps = []
     foreach lib_name : lib_torch_list
-        tdeps += cppc.find_library(lib_name, dirs: [LIB_TORCH_LIB_PATH])
+        _lib = cppc.find_library(lib_name, dirs: [LIB_TORCH_LIB_PATH], required: false)
+        if _lib.found()
+            tdeps += _lib
+        endif
     endforeach
