diff --git a/CppCore/rgpot/rpc/meson.build b/CppCore/rgpot/rpc/meson.build
index e3c7393..e41eb65 100644
--- a/CppCore/rgpot/rpc/meson.build
+++ b/CppCore/rgpot/rpc/meson.build
@@ -1,15 +1,15 @@
 # Find dependencies and tools
 capnp_rpc_dep = dependency('capnp-rpc')
-capnpc = find_program('capnpc')
+capnp = find_program('capnp')

 gen_rpc = custom_target(
     'gen_rpc',
     input: 'Potentials.capnp',
     output: ['Potentials.capnp.c++', 'Potentials.capnp.h'],
     command: [
-        capnpc,
-        '-o',
-        'c++:@OUTDIR@',
+        capnp,
+        'compile',
+        '-oc++:@OUTDIR@',
         '--src-prefix=' + meson.current_source_dir(),
         '@INPUT@',
     ],
